src/actions/manager.tsx

```diff
@@ -173,13 +235,37 @@ export class ActionManager {
     return null;
   };
 
-  isActionEnabled = (action: Action) => {
-    const elements = this.getElementsIncludingDeleted();
+  isActionEnabled = (
```

Comment:
You might want to plug in `isActionEnabled` check into `_openContextMenu` function in order to make sure disabled by `registerDisableFn` call actions are not added into `ContextMenu`. Otherwise we will see those actions available in the context menu but click won't do anything.